projects/plugins/jetpack/_inc/lib/class-jetpack-ai-helper.php

```diff
@@ -332,6 +332,7 @@ public static function get_ai_assistance_feature() {
 				'requests-count'       => $requests_count,
 				'requests-limit'       => $requests_limit,
 				'site-require-upgrade' => $require_upgrade,
+				'upgrade-prompt-type'  => wpcom_is_vip( $blog_id ) ? 'vip' : 'default',
```

Comment:
I'm not totally convinced about the parameter name :-/. Can we add something about the site type? Maybe just `is-vip`?
```suggestion
				'is-vip'  => wpcom_is_vip( $blog_id ),
```

Reply:
I so... you'll have to update the rest of the code

Reply:
I thought about sending the site type, but it seemed a bit off, kind of unrelated to the purpose of the endpoint. The choice for `upgrade-prompt-type` was a way to flag different types of prompts, like `vip`, `default`, `atomic-trial`, `jetpack-bonus-trial` or whatever we can come up with.

Maybe the `vip` mention is too specific, and we could call it `no-upgrade-button` option, for example.

What do you think?

Reply:
IMO, in this context, setting conditions about the UI (show or not show an upgrade button) doesn't fit very well. The data concerns the `AI Assistant` feature: `requests-count`, `request-limit`, and even if the site requires an upgrade. 

The most straightforward option here is to change the type of `'site-require-upgrade'`. Instead of being a boolean, it can be a string that defines the upgrade type: `wpcom`, `jetpack`, `vip`, etc.

I think that's a nice approach.